static/app/views/replays/detail/focusArea.tsx

```diff
@@ -55,8 +56,8 @@ function ActiveTab({
           />
         </div>
       ) : null;
-    case 'errors':
-      return <div id="errors">TODO: Add an errors view</div>;
```

Comment:
This drops the `id="errors"` bit which i think is totally fine. I was finding it annoying and want to drop it for the other tabs too. 

Billy has a fix here related to this stuff: https://github.com/getsentry/sentry/pull/34197/files#r864271870

I just wanted to call it out because it looks like an accidental change but imo it's fine to keep.